test/test.make

```diff
@@ -110,6 +110,7 @@ space:= $(empty) $(empty)
 RUN_E2E = KUBECONFIG=`pwd`/_work/$(CLUSTER)/kube.config \
 	REPO_ROOT=`pwd` \
 	CLUSTER=$(CLUSTER) \
+	TEST_DEPLOYMENTMODE=$(shell source test/test-config.sh; echo $$TEST_DEPLOYMENTMODE) \
```

Comment:
I think we have an extra `$` when echoing `TEST_DEPLOYMENTMODE` variable, aren't we?
`echo $$TEST_DEPLOYMENTMODE` -> `echo $TEST_DEPLOYMENTMODE`?